padinfo/boardgenerator.py

```diff
@@ -0,0 +1,88 @@
+class BoardGenerator(object):
+    """Generate a Dawnglare link with optional inversion."""
+
+    allowed_letters = "RGBLDHXJMPZ "
+    inversion = "-1"
+    dawnglare_link = "https://pad.dawnglare.com/?patt={}&showfill=1"
+
+    def __init__(self, board):
+        self.board = board.split()
+        self.invalid_board = False
+        self.invert = self.inversion_check()
+        self.generate_clean_board()
+        self.validate_orbs()
+        self.board_length = len(self.clean_board)
+        self.eval_board_size()
+        self.generate_link()
+
+    def inversion_check(self):
```

Comment:
This whole function can be super well condensed into the one line of
```py
return BoardGenerator.inversion in self.board
```

And at that point, we don't even need to keep it as a function! It's so short and clear that we can just put it up into the \_\_init\_\_ :)